addons/editor-devtools/DevTools.js

```diff
@@ -8,6 +8,9 @@ export default class DevTools {
     this.addon = addon;
     this.msg = msg;
     this.m = m;
+    /**
+     * @type {VirtualMachine}
+     */
```

Comment:
This comment should actually be removed - the variable it refers to was removed in #4817.